base/statistics.jl

```diff
@@ -66,13 +90,26 @@ function mean!(R::AbstractArray, A::AbstractArray)
 end
 
 """
-    mean(v; dims)
+    mean(A::AbstractArray; dims)
 
-Compute the mean of whole array `v`, or optionally along the given dimensions.
+Compute the mean of an array over the given dimensions.
```

Comment:
I'm not sure this reformulation is an improvement

Reply:
Note that I've added a separate docstring (above) for the simpler method which doesn't take `dims` and isn't restricted to `AbstractArray`. This is consistent with other reductions. What would you like to change?